### DISCUSSION TITLE
Generate PDF of a form

### DISCUSSION THREAD

**Discussion by nourdar** [filamentphp/filament#13726] [Ideas]
Hello, I'm working on a project and encountering an issue. The current task is to send an email to the customer with a filled form attached as a PDF. I need to create a new blade, structure the form, and replace the data for each instance. Is there a way to export the form to a PDF while maintaining the same layout as the view page?

**Comment by RoseRiyadh** [comment]
> Hello, I'm working on a project and encountering an issue. The current task is to send an email to the customer with a filled form attached as a PDF. I need to create a new blade, structure the form, and replace the data for each instance. Is there a way to export the form to a PDF while maintaining the same layout as the view page?

Have you found the solution?
